integrationTests/ts/package.json

```diff
@@ -9,6 +9,8 @@
     "typescript-4.1": "npm:typescript@4.1.x",
     "typescript-4.2": "npm:typescript@4.2.x",
     "typescript-4.3": "npm:typescript@4.3.x",
-    "typescript-4.4": "npm:typescript@4.4.x"
+    "typescript-4.4": "npm:typescript@4.4.x",
+    "typescript-4.5": "npm:typescript@4.5.1-rc",
+    "typescript-next": "npm:typescript@next"
```

Comment:
please don't do it next, if TS releases a new version it will break CI.

Reply:
> please don't add next, if TS releases a new version it will break CI.

No problem, but I was just doing what you told me to, keep in mind that the new typescript ESM is no longer in 4.5.1, it's in @ next, see https://devblogs.microsoft.com/typescript/announcing-typescript-4-5-rc/#beta-delta